<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="18" hostname="ggn001345" failures="5" tests="10" name="Auth" time="9.903" errors="0" timestamp="18 May 2021 10:52:50 GMT">
  <testcase classname="global_api.global_api.testCases.SignUp" name="invalidSignup" time="2.244"/>
  <testcase classname="global_api.global_api.testCases.SignUp" name="validSignUp" time="1.102"/>
  <testcase classname="global_api.global_api.testCases.UserInfo" name="userInfo" time="0.853">
    <failure type="java.lang.AssertionError" message="expected [401] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [401] but found [200]
at global_api.global_api.testCases.UserInfo.userInfo(UserInfo.java:32)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- userInfo -->
  <testcase classname="global_api.global_api.testCases.Forgot_password" name="change_pwd" time="0.74">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at global_api.global_api.testCases.Forgot_password.change_pwd(Forgot_password.java:42)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- change_pwd -->
  <testcase classname="global_api.global_api.testCases.Forgot_password" name="forgot_pwd" time="0.552"/>
  <testcase classname="global_api.global_api.testCases.Login_admin" name="login_admin" time="0.548">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at global_api.global_api.testCases.Login_admin.login_admin(Login_admin.java:33)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- login_admin -->
  <testcase classname="global_api.global_api.testCases.Otp" name="otp_send" time="0.754">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at global_api.global_api.testCases.Otp.otp_send(Otp.java:40)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- otp_send -->
  <testcase classname="global_api.global_api.testCases.Refresh" name="refresh" time="1.531">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at global_api.global_api.testCases.Refresh.refresh(Refresh.java:27)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- refresh -->
  <testcase classname="global_api.global_api.testCases.Login_user" name="invaliadLogin" time="0.699"/>
  <testcase classname="global_api.global_api.testCases.Login_user" name="validlogin" time="0.568"/>
  <testcase classname="global_api.global_api.base.TestBase" name="@AfterSuite endReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite endReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@AfterSuite endReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite endReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@AfterSuite endReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite endReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@AfterSuite endReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite endReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@AfterSuite endReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite endReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@AfterSuite endReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite endReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeTest setReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeTest setReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeTest setReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeTest setReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeTest setReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setReport -->
  <testcase classname="global_api.global_api.base.TestBase" name="@BeforeTest setReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setReport -->
</testsuite> <!-- Auth -->
